seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-elasticsearch7/pom.xml

```diff
@@ -27,7 +27,11 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>seatunnel-connector-flink-elasticsearch</artifactId>
+    <artifactId>seatunnel-connector-flink-elasticsearch7</artifactId>
+
+    <properties>
+        <elasticsearch7.version>7.5.1</elasticsearch7.version>
+    </properties>
```

Comment:
The version number needs to be declared in the root-pom

Reply:
Done